CONTRIBUTING.md

```diff
@@ -22,14 +22,8 @@ For more information, please see the Eclipse Committer Handbook:
 https://www.eclipse.org/projects/handbook/#resources-commit
 
 ## Code Style Guide
-* Use [Black](https://black.readthedocs.io/) to format your code.
-* Use [isort](https://isort.readthedocs.io/) to sort imports.
-* Use [pydocstyle](https://pydocstyle.readthedocs.io/) to check for PEP-8 style issues.
-* Use [mypy](https://mypy.readthedocs.io/) to check for type errors.
-* Use [flake8](https://flake8.readthedocs.io/) to check for style issues.
-* Use [pylint](https://pylint.readthedocs.io/) to check for code quality issues.
-* Above and other tools will run automatically if you install
- [pre-commit](https://pre-commit.com/) using the instructions below.
+* clang-format will format your code according to the established coding style
```

Comment:
we are using more linters and also pre-commit, please add more details here

Reply:
Done